projects/packages/forms/src/blocks/contact-form/components/jetpack-integrations-modal/integration-card/integration-card-header.js

```diff
@@ -96,13 +96,13 @@ const IntegrationCardHeader = ( {
 							trackEventName={ cardData.trackEventName }
 						/>
 					) }
-					{ ( isActive || isConnected ) && showHeaderToggle && (
+					{ showHeaderToggle && (
 						<Tooltip text={ getTooltipText( headerToggleValue ) }>
 							<span className="integration-card__toggle-tooltip-wrapper">
 								<ToggleControl
-									checked={ headerToggleValue }
+									checked={ headerToggleValue && ( isActive || isConnected ) }
 									onChange={ handleToggleChange }
-									disabled={ ! isHeaderToggleEnabled }
+									disabled={ ! isHeaderToggleEnabled || ! ( isActive || isConnected ) }
```

Comment:
This is a generic improvement that was needed for salesforce but improves the card header. Before: if plugin was not active or service not connected, we would not show a toggle to enable the integration for this form. Now: we will show the toggle, but it will be disabled/off.